heatmap/src/window.rs

```diff
@@ -20,7 +20,7 @@ impl<Value, Count> Window<Value, Count> {
         self.stop
     }
 
-    pub fn histogram(&self) -> &Histogram<Value, Count> {
+    pub fn histogram(&self) -> &Histogram {
```

Comment:
If you have a reference internally it's generally better to pass the lifetime on instead of shortening it 
```suggestion
    pub fn histogram(&self) -> &'a Histogram {
```

Reply:
Fixed.